api/src/main/resources/fireCloud.yaml

```diff
@@ -317,18 +279,39 @@ definitions:
   Profile:
       type: object
       properties:
-        name:
+        firstName:
           type: string
-          description: User's full name
-        email:
```

Comment:
Should we also get a user's `@researchallofus` email here? (along with their contact email?)

Reply:
This is the FC API; it only has contactEmail. (The other e-mail is implied by the account you're using.)

Our /me endpoint will return a Profile object with both (see workbench.yaml).

Reply:
Yup, I noticed that there. Sounds good.